src/ConfigParser.h

```diff
@@ -232,7 +232,10 @@ class ConfigParser
     static enum ParsingStates {atParseKey, atParseValue} KvPairState_; ///< Parsing state while parsing kv-pair tokens
 };
 
-int parseConfigFile(const char *file_name);
+namespace Configuration {
+/// interprets (and partially applies) squid.conf or equivalent configuration
+void Parse();
```

Comment:
Will do after current configuration-related PR land. I will move quickly with these upcoming changes, but I am at your mercy as far as unblocking that work (by re-reviewing posted PRs) is concerned...

FWIW, I do not recommend working on configuration code in the nearest future because doing so will increase the number of conflicts we would need to resolve.